website/README.md

```diff
@@ -1 +1,145 @@
-We need a website, let's start putting content in here :)
+Website will be customized using docusaurus:
+
+## Getting Started
+
+In your project repo, make sure you have a `docs` folder containing all of your documentation files written in markdown. Create a `website` folder inside which you will install and run docusaurus. If you wish, you can also include a `blog` folder in your project repo at the same level as the `docs` folder for blog posts.
```

Comment:
this seems to be very generic docusaurus how to install. But since you already went through it, it's not very useful anymore.